src/common/dsp/SurgeVoice.cpp

```diff
@@ -191,7 +191,9 @@ SurgeVoice::SurgeVoice(SurgeStorage *storage, SurgeSceneStorage *oscene, pdata *
     state.key = key;
     state.keyRetuningForKey = -1000;
     state.channel = channel;
+
     state.voiceOrderAtCreate = voiceOrder;
+    state.voiceMax = storage->getPatch().polylimit.val.i;
```

Comment:
Is there any reason we call this 'voiceMax' vs 'polylimit'? We call it 'polylimit' everywhere else in the code so the new name is a potential source of confusion.

Reply:
Good point, the undecided name in the Lua state table was guiding me here but makes more sense to keep it consistent.

Reply:
Yeah cool, so polylimit it is!